### PR TITLE
MGMT-15505: assisted-installer-agent 2.9 config

### PR DESCRIPTION
**Pull request by adriengentil**

### CODE REVIEW

File: ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-release-ocm-2.9.yaml [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    env:
+      DOCKERFILE_IMAGE_PAIRS: |
+        Dockerfile.assisted_installer_agent , quay.io/edge-infrastructure/assisted-installer-agent-multi:ocm-2.9
+    workflow: assisted-tools-multi-arch
+- as: mirror-nightly-image
+  cluster: build01
+  cron: '@daily'
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: assisted-installer-agent
```
Comment (adriengentil) [inline ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-release-ocm-2.9.yaml:94]: need to set version here

File: ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-release-ocm-2.9.yaml [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+test_binary_build_commands: |
+  go mod vendor
+  mkdir /.cache && chmod 775 -R /.cache ${GOPATH}
+tests:
+- as: publish-multi-arch-images
+  cluster: build01
```
[the remark this answers is not in the document]
Comment (adriengentil) [reply ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-release-ocm-2.9.yaml:81]: I will remove all those cluster thingy

[the remark this answers is not in the document]
Comment (adriengentil) [reply ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-release-ocm-2.9.yaml]: done

**Review by osherdp — APPROVED** [review]

**Review by eliorerz — APPROVED** [review]

**Pull request merged**